src/pages/Index.vue

```diff
@@ -711,13 +744,15 @@
 		.examples {
 			display         : flex;
 			justify-content : space-around;
-			width           : 80%;
 			max-width       : 1100px;
 
-			& > div {
-				width : 40%;
-			}
+			@media screen and (max-width:1500px){
+				width: 95%;
+			}			
 
+			@media screen and (max-width:700px){
+				width: 100%;
+			}
```

Comment:
On line 870, the `code:not(.javascript)` rules has been affected and now the line height it too big compared than before

Reply:
Not sure what you mean here. On my screen the `code:not(.javascript)` elements look the exact same as they did on [autonumeric.org](www.autonumeric.org)